Sources/ComposableArchitecture/Documentation.docc/Articles/TreeBasedNavigation.md

```diff
@@ -357,42 +357,10 @@ seemingly disparate forms of navigation can be unified under a single style of A
 
 #### Backwards compatible availability
 
-Depending on your deployment target, certain APIs may be unavailable. For example, if you target
```

Comment:
This change is not related the rest of the PR, but I just realized that our backwards compatibility advice is now outdated since we did back port `navigationDestination(item:)` in SwiftUINavigationCore. Also the `NavigationLink` code wasn't quite right because it didn't have an `onNavigate` action closure.